config/v1/0000_10_config-operator_01_infrastructure-CustomNoUpgrade.crd.yaml

```diff
@@ -51,6 +51,13 @@ spec:
                     aws:
                       description: AWS contains settings specific to the Amazon Web Services infrastructure provider.
                       properties:
+                        customDNS:
+                          description: CustomDNS indicates whether a custom DNS config is enabled or disabled.
```

Comment:
It would be helpful to add a description that describes a bit more about what the config item indicates. In this case, it could be :
"CustomDNS indicates whether the customer wants to use their own DNS solution in place of the cloud provided solution. Its value is set during install time and cannot be changed after. The default value is "Disabled".